content/desktop/hardened-desktop/_index.md

```diff
@@ -31,38 +31,23 @@ grid:
-- Enhanced Container Isolation (ECI), a setting that instantly enhances security by preventing containers from running as root in Docker Desktop’s Linux VM and ensures that any configurations set using Settings Management cannot be bypassed or modified by containers.
-- Registry Access Management (RAM), which allows admins to control the registries developers can access.
-- Image Access Management (IAM), which gives admins control over which images developers can pull from Docker Hub.
-- Air-gapped containers, which restricts containers from accessing unwanted network resources.
-
 ### How does it help my organisation?
 
-Hardened Desktop features work independently but collectively to create a defense-in-depth strategy, safeguarding developer workstations against potential attacks across various functional layers, such as configuring Docker Desktop, pulling container images, and running container images. This multi-layered defense approach ensures comprehensive security.
+Hardened Desktop features work independently but collectively to create a defense-in-depth strategy, safeguarding developer workstations against potential attacks across various functional layers, such as configuring Docker Desktop, pulling container images, and running container images. This multi-layered defense approach ensures comprehensive security. It helps mitigate against threats such as:
 
-It helps mitigate against threats such as:
- - **Malware and supply chain attacks:** RAM and IAM prevent developers from accessing certain container registries and image types, significantly lowering the risk of malicious payloads. Additionally, ECI restricts the impact of containers with malicious payloads by running them without root privileges inside a Linux user namespace.
- - **Lateral movement:** Air gapped containers allows admins to configure network access restrictions for containers, thereby preventing malicious containers from performing lateral movement within the organization's network.
- - **Insider threats:** Settings Management configures and locks various Docker Desktop settings, such as proxy settings, ECI, and prevents exposure of the Docker API. This helps admins enforce company policies and prevents developers from introducing insecure configurations, intentionally or unintentionally.
+ - Malware and supply chain attacks: Registry Access Management and Image Access Management prevent developers from accessing certain container registries and image types, significantly lowering the risk of malicious payloads. Additionally, ECI restricts the impact of containers with malicious payloads by running them without root privileges inside a Linux user namespace.
+ - Lateral movement: Air gapped containers lets administrators to configure network access restrictions for containers, thereby preventing malicious containers from performing lateral movement within the organization's network.
```

Comment:
I've seen this without the hyphen as well but just a suggestion: 
```suggestion
 - Lateral movement: Air-gapped containers lets administrators to configure network access restrictions for containers, thereby preventing malicious containers from performing lateral movement within the organization's network.
```

Reply:
```suggestion
- Lateral movement: Air-gapped containers lets administrators configure network access restrictions for containers, thereby preventing malicious containers from performing lateral movement within the organization's network.
```